src/Packagist/WebBundle/Entity/SecurityAdvisory.php

```diff
@@ -79,9 +84,11 @@ public function updateAdvisory(RemoteSecurityAdvisory $advisory): void
             $this->title !== $advisory->getTitle() ||
             $this->link !== $advisory->getLink() ||
             $this->cve !== $advisory->getCve() ||
-            $this->affectedVersions !== $advisory->getAffectedVersions()
+            $this->affectedVersions !== $advisory->getAffectedVersions() ||
+            $this->reportedAt !== $advisory->getDate()
```

Comment:
using strict comparison on a DateTime instead is wrong. It will always report them being different as they won't be the same object. You should use `!=` to compare them by value.